manuscript/draft.Rmd

```diff
@@ -574,20 +574,20 @@ The most commonly used methods for forecasting future biodiversity, SDMs and
 macroecological models, both produced worse forecasts than time-series models
 and simple baselines. This weakness suggests that predictions about future
 biodiversity change should be viewed with skepticism unless the underlying
-models have been validated temporally, via hindcasting and comparison with
+models are temporally validated, via hindcasting and comparison with
 simple baselines. Since site-level richness is relatively stable, spatial
-validation is not enough: a model can have high accuracy across spatial
-gradients without being able to predict changes over time. This gap between
-spatial and temporal accuracy is known to be important for species-level
+validation is not enough: a model be accurate across spatial gradients while
+being unable to predict changes over time. This gap between
+spatial and temporal accuracy is important for species-level
 predictions [@rapacciuolo2012; @oedekoven2017]; our results indicate that it is
-substantial for higher-level patterns like richness as well. SDMs' poor temporal
-predictions are particularly sobering, as these models have been one of the main
-foundations for estimates of the predicted loss of biodiversity to climate
-change over the past decade or so [@thomas2004; @thuiller2011; @urban2015]. Our
+substantial for higher-level patterns like richness as well. The poor quality 
+temporal predictions of SDM's are particularly sobering because of their importance
+over the last decade or so for predicting biodiversity loss as a response to 
```

Comment:
Let's also get rid of "or so" here.

Reply:
How about "over the last two decades for predicting..." since we have an '04 reference in there.

Reply:
Sounds good.